src/main/java/com/coreos/jetcd/internal/impl/WatchImpl.java

```diff
@@ -149,7 +145,7 @@ private synchronized void cancelWatcher(long id) {
 
   private synchronized StreamObserver<WatchRequest> getGrpcWatchStreamObserver() {
     if (this.grpcWatchStreamObserver == null) {
-      this.grpcWatchStreamObserver = this.watchStub.watch(this.createWatchStreamObserver());
+      this.grpcWatchStreamObserver = stub.watch(this.createWatchStreamObserver());
```

Comment:
This is an unintended change caused by an intermediate refactor so I'll bring them back. 

Beside the use of ``this`` when it is needed (disambiguate variable reference etc.) using it everywhere is more a style choice that a good/bad practice. I personally find that it does not add much value and definitively not useful for methods but again is a style choice.

To make the source code consistent you'd need to add a check-style rule as today the ``this`` is not used consistently.